src/components/header/_header.scss

```diff
@@ -60,6 +60,12 @@ header {
             li {
                 margin: 0;
                 cursor: pointer;
+                a:hover {
+                    color: $footer;
+                }
+                .active {
```

Comment:
Not consistant to rest of the designs, I recommend to change the color to our yellow in the app instead of adding the border

Reply:
done